libnavui-voice/src/main/res/values-pl/strings.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <!-- Indicates voice instructions are muted -->
-    <string name="mapbox_muted">Wyciszony</string>
+    <string name="mapbox_muted">Dźwięk wył.</string>
 
     <!-- Indicates voice instructions are unmuted -->
-    <string name="mapbox_unmuted">Niewyciszony</string>
+    <string name="mapbox_unmuted">Dźwięk wł.</string>
 </resources>
```

Comment:
https://github.com/mapbox/mapbox-navigation-android/assets/16925074/e4b41323-b00a-4934-9b1c-eec5037b1abf

Reply:
@azarovalex please have a look in here.
It might be better translation then the one I've provided.
```
    mapbox_muted = Dźwięk wył.
    mapbox_unmuted = Dźwięk wł.
```